test/test.googleauth.ts

```diff
@@ -402,18 +403,30 @@ describe('googleauth', () => {
   });
 
   it('getApplicationCredentialsFromFilePath should not error on valid symlink', async () => {
+    if (onWindows()) {
+      // git does not create symlinks on Windows
+      return;
+    }
     await auth._getApplicationCredentialsFromFilePath(
       './test/fixtures/goodlink'
     );
   });
 
   it('getApplicationCredentialsFromFilePath should error on invalid symlink', async () => {
+    if (onWindows()) {
```

Comment:
I mean, if windows doesn't create the symlink, it shouldn't matter right?

Reply:
Right! Removed this